tests/ui/compile-fail/init/colon_instead_of_arrow.stderr

```diff
@@ -1,20 +1,20 @@
 error[E0308]: mismatched types
-  --> tests/ui/compile-fail/init/colon_instead_of_arrow.rs:21:9
-   |
-14 |     fn new() -> impl PinInit<Self> {
-   |                 ------------------ the found opaque type
+ --> tests/ui/compile-fail/init/colon_instead_of_arrow.rs:21:9
+  |
+  14 |     fn new() -> impl PinInit<Self> {
+     |                 ------------------ the found opaque type
```

Comment:
there are some weird changes in here, such as this one, it removes two spaces in the first line with the `|`...

any ideas @nbdd0121? maybe I'll have to update the macrotest & trybuild crates

Reply:
Hah, I skipped "whitespace only changes" :) this indeed looks like an update needed.